build.sbt

```diff
@@ -102,10 +97,15 @@ def playJsonOpsCommon(scalacVersion: String, includePlayVersion: String): Projec
 lazy val `play25-json-ops-common-211` = playJsonOpsCommon(Scala_2_11, Play_2_5)
 lazy val `play26-json-ops-common-211` = playJsonOpsCommon(Scala_2_11, Play_2_6)
 lazy val `play26-json-ops-common-212` = playJsonOpsCommon(Scala_2_12, Play_2_6)
+lazy val `play27-json-ops-common-211` = playJsonOpsCommon(Scala_2_11, Play_2_7)
+lazy val `play27-json-ops-common-212` = playJsonOpsCommon(Scala_2_12, Play_2_7)
```

Comment:
Trying to make sure I understand the structure here.  My interpretation is that the 2.13 version simply doesn't make use of playJsonOpsCommon -- correct?

Reply:
Yup. The collections code is different enough that it isn't source code compatible, so I created a separate copy of the source code, formatted for Scala 2.13